receipt_dynamo/receipt_dynamo/entities/geometry_entity.py

```
@@ -0,0 +1,726 @@
+            b1 = c - x_new_px
+
+            a21 = g * y_new_px - d
+            a22 = h * y_new_px - e
+            b2 = f - y_new_px
+
+            # Solve via determinant
+            det = a11 * a22 - a12 * a21
+            if abs(det) < 1e-12:
+                raise ValueError(
+                    "Inverse perspective transform is singular for this corner"
+                )
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)